### PR TITLE
Change response id, username to createUser

### PR DESCRIPTION
**Pull request by ealexisaraujo**

### CODE REVIEW

File: components/users/usersService.js [hunk 1]
```diff
@@ -99,7 +99,8 @@ class usersService {
       this.collection,
       new UserModel({ ...user, password: hashedPassword, username: username })
     );
-    return createUserId, username;
+
+    return { createUserId, username };
```
Comment (hyfi06) [inline components/users/usersService.js:103]: Does it not affect another function?

File: components/users/routes.js [hunk 2]
```diff
@@ -89,9 +89,15 @@ function usersApi(app) {
...
+        const { createUserId: id, username } = await usersService.createUser({
+          user,
+        });
+
         res.status(201).json({
-          data: createUserId,
+          data: {
+            id,
+            username,
+          },
```
Comment (hyfi06) [inline components/users/routes.js:100]: it isn't a typical answer.

File: components/users/usersService.js (hunk 1 above)
Comment (ealexisaraujo) [reply components/users/usersService.js:103]: Answer is not, I checked service createUser

File: components/users/routes.js (hunk 2 above)
Comment (ealexisaraujo) [reply components/users/routes.js]: I changed for this:

```json
res.status(201).json({
          data: {
            _id: id,
          },
          message: `User ${username} created`,
        });
```

**Pull request merged**